gensrc/script/functions.py

```diff
@@ -630,6 +630,9 @@
     [91000, 'sub_bitmap', 'BITMAP', ['BITMAP', 'BIGINT', 'BIGINT'], 'BitmapFunctions::sub_bitmap', False],
     [91001, 'bitmap_subset_limit', 'BITMAP', ['BITMAP', 'BIGINT', 'BIGINT'], 'BitmapFunctions::bitmap_subset_limit', False],
     [91002, 'bitmap_subset_in_range', 'BITMAP', ['BITMAP', 'BIGINT', 'BIGINT'], 'BitmapFunctions::bitmap_subset_in_range', False],
+    [91003, 'bitmap_to_binary', 'VARBINARY', ['BITMAP'], 'BitmapFunctions::bitmap_to_binary', False],
+    [91004, 'bitmap_from_binary', 'BITMAP', ['VARBINARY'], 'BitmapFunctions::bitmap_from_binary', False],
+    [91005, 'bitmap_from_binary', 'BITMAP', ['VARCHAR'], 'BitmapFunctions::bitmap_from_binary', False],
```

Comment:
Why there are two functions with different input types?
Because the input should be a valid binary which is generated by bitmap_to_binary. So the input should be a varchar type.
Varchar means that the content will be valid characters, but the bitmap deserialized binary can not guarantee it.
So you need to remove the varchar input function to make it reasonable.

Reply:
If not, bitmap_from_binary(varchar) will be implicitly converted to bitmap_from_binary(cast(varchar as binary)). However, we currently do not support cast(varhcar as binary). In theory, varchar can be converted to binary, and it cannot be converted directly. In this way, users cannot use it directly on the import.

Reply:
n order to be consistent with base64_to_bitmap, null is returned uniformly.

Reply:
Fixed